files/prometheus_alerts.yml

```diff
@@ -134,3 +134,13 @@
     for: 10m
     labels:
       severity: warning
+
+  - "alert": "ElasticsearchOperatorCSVNotSuccessful"
+    "annotations":
+      "message": "Elasticsearch Operator CSV has not reconciled succesfully."
+      "summary": "Elasticsearch Operator CSV Not Successful"
+    "expr": |
+      csv_succeeded{name =~ "elasticsearch-operator.*"} == 0
```

Comment:
does this expression simply check the status of the csv to denote if it was successful?

Reply:
iirc, csv_succeeded == 0 is the failed phase